superset/datasets/commands/importers/v1/utils.py

```diff
@@ -148,6 +156,26 @@ def import_dataset(
     return dataset
 
 
+def extract_tags(config: Dict[str, Any]) -> Tuple[Dict[str, Any], Optional[List[str]]]:
+    tags = None
+    if "tags" in config.keys() and feature_flag_manager.is_feature_enabled(
+        "TAGGING_SYSTEM"
+    ):
+        tags = config.pop("tags")
+
+    return config, tags
+
+
+def import_tags(dataset: SqlaTable, existing: bool, tags: Optional[List[str]]) -> None:
```

Comment:
Looking at this method, it's only doing something if `tags` has value in it, so why is `tags` `Optional` then? Can we just mark it as required and not call this method if tag if empty for example?